src/App.js

```diff
@@ -1,7 +1,7 @@
 import Header from "./Components/Header";
 import Login from "./Components/Login";
 import Main from "./Components/Main";
-import Projects from "./Components/Projects";
+import Projects from "./Pages/Projects";
```

Comment:
Could you please rollback this as it's not in the scope of the PR / Issue ?  

Also, If Projects is put into pages, then Login & others will also be need to be added into pages. (Which is currently worked upon by other contributors) 

So, we can do this some other time.

Reply:
Done